src/isc-dhcp/patch/0007-Add-enable-use-sockets-to-configure-flags-in-debian-.patch

```diff
@@ -0,0 +1,28 @@
+From ab779f1d59f27f66e6e6bea89287c810e0a10a0f Mon Sep 17 00:00:00 2001
+From: Joe LeVeque <jolevequ@microsoft.com>
+Date: Wed, 15 May 2019 23:48:08 +0000
+Subject: [PATCH 1/3] Add --enable-use-sockets to configure flags in
+ debian/rules
+
+This defines USE_SOCKETS at compile time which forces dhcrelay
+to create and bind separate sockets to each interface
+---
+ debian/rules | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/debian/rules b/debian/rules
+index 4184716..46d6527 100755
+--- a/debian/rules
++++ b/debian/rules
+@@ -23,7 +23,7 @@ CFLAGS+=-D_PATH_DHCLIENT_CONF='\"/etc/dhcp/dhclient.conf\"'
+ CFLAGS+=-D_PATH_DHCLIENT_DB='\"$(LEASE_PATH)/dhclient.leases\"'
+ CFLAGS+=-D_PATH_DHCLIENT6_DB='\"$(LEASE_PATH)/dhclient6.leases\"'
+ 
+-CONFFLAGS=--prefix=/usr --enable-log-pid --enable-paranoia
++CONFFLAGS=--prefix=/usr --enable-log-pid --enable-paranoia --enable-use-sockets
```

Comment:
>ONFFLAGS=--prefix=/usr --enable-log-pid --enable-paranoia --enable-use-sockets [](start = 2, length = 78)

I meant patching the override_dh_auto_build

Reply:
What's the difference? `override_dh_auto_build` references `CONFFLAGS` a few lines below. I feel it's better to add it here so it gets applied everywhere `CONFFLAGS` is used, rather than adding it in multiple places.

Reply:
sorry, I meant override_dh_auto_configure.

---
In reply to: [284958614](https://github.com/Azure/sonic-buildimage/pull/2857#discussion_r284958614) [](ancestors = 284958614)

Reply:
The problem with `override_dh_auto_build` is that in this rules file, they have turned `override_dh_auto_build` into a no-op. Instead of calling `dh_auto_configure` here, they instead directly call `./configure` on three separate occasions in `override_dh_auto_build`. Thus, we cannot take that approach here.